padertorch/configurable.py

```diff
@@ -191,6 +197,44 @@ class Configurable:
     Another use case for this behaviour are depended config entries (e.g.
     NN input size depends on selected input features).
 
+    Some modules need unitilized classes or functions as an input.
+    Therefore, one can use the partial key instead of the factory key.
+    All defined kwargs will overwrite the defaults without calling the function
+    or initializing the class using partial.
+    One example is SpeechBrain which requires the activity to be not
+    initialized at the class input.
```

Comment:
Mention, that we support functions as arguments, but to support json/yaml serialization, we recommend the partial?